test/App.test.js

```diff
@@ -0,0 +1,49 @@
+import React from 'react';
+import ReactDOM from 'react-dom';
+import TestRenderer from 'react-test-renderer';
+import { mount, shallow } from 'enzyme';
+import axios from 'axios';
+import MockAdapter from 'axios-mock-adapter';
+import App from "../src/App";
+
+it('should render without crashing', () => {
+  const div = document.createElement('div');
+  ReactDOM.render(<App />, div);
+  ReactDOM.unmountComponentAtNode(div);
+});
+
+describe('App Snapshot', () => {
+  test('renders', () => {
+    const component = TestRenderer.create(<App />);
+    let tree = component.toJSON();
+    expect(tree).toMatchSnapshot();
+  });
+});
+
+describe('should render App component', () => {
+  const element = mount(<App />);
+  test('Snapshot test with default props', () => {
+    expect(element).toMatchSnapshot();
+  });
+});
+
+describe('should fetch data', () => {
+  it('should update state with response data', () => {
+    const url = 'http://react-cdp-api.herokuapp.com/movies?search=&searchBy=title';
+    const mock = new MockAdapter(axios);
+    mock
```

Comment:
It's a good practice to remove mocking behavior (`mock.restore()`) after given test is finished - https://jestjs.io/docs/en/setup-teardown

Reply:
Done.